packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.js

```diff
@@ -18,7 +19,9 @@ const propTypes = {
   /** Default hyperlink location */
   href: PropTypes.string,
   /** Additional props are spread to the container component */
-  '': PropTypes.any
+  '': PropTypes.any,
+  /** Callback for click event */
+  onClick: PropTypes.func
```

Comment:
Also need to update the props in the DropdownItem.d.ts file?

Reply:
I will add the onClick to the ts file